packages/shared/src/utils/constants/barriers.ts

```diff
@@ -14,6 +14,8 @@ export const CONTRACT_SHADES = {
     ASIAND: 'BELOW',
     MULTUP: 'ABOVE',
     MULTDOWN: 'BELOW',
+    TURBOSLONG: 'ABOVE',
```

Comment:
@maryia-deriv Not related to this PR, But would be nice if you could add `as const` to all variables in this file to improve TS 🙏🏻

Reply:
@farzin-deriv unfortunately, I have to ignore it because of it's out of this PR scope like you mentioned.